xml/Microsoft.Extensions.Logging.Console/ConsoleLoggerOptions.xml

```diff
@@ -232,7 +232,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets indication whether or not UTC timezone should be used to for timestamps in logging messages. Defaults to <see langword="false" />.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <summary>Gets or sets a value that indicates whether or not UTC timezone should be used to for timestamps in logging messages. Defaults to <see langword="false" />.</summary>
        <value><see langword="true" /> to use UTC timezone; <see langword="false" /> otherwise.</value>
```